docs/examples/10 minutes to QCoDeS.ipynb

```diff
@@ -0,0 +1,966 @@
lpYdy4cRQKBSoqKgbd4xgxYhy76OkJeO+9DrZu7aBQCAd/4BPiWCUsf/zjH/noo4/47ne/+4nmHePI4QQhGjoWwp62azcmXyvtNs2tceJ3NEEBUdVwDjzJN01tPQpUzINrTTDVu13CYglJiZKO1IprqBRqz1gCHOU/qd2aAh0HYfLCero+hiELQqh0tCa9rW+2W1+38jjWD89T95Ll2t1J+xklyqL+3LmbgHJPODxKExDfkA7d3vcihd3XBERXTT/99DTPbRdQLWA4UW5a4zJliZW0xEkiyQnICZg+UfLypi5aZ35GhYkgySCoAJJ66DLNUZL6lfHUKymgzFfVwZMCUp4g4UnSHjzxxlZad+xWRFSn4rWZs8y6G5c7t8CjRJGNQnbIv1LxKfpOrV+/nrlz5xalsz3rrLNYvnw5S5Ysob29nenTp/PYY4/Zk/qGhoYiC8hAqK+vL4qJyOVyvPbaa9x999322tq1a5k3bx6JRII5c+awcuVKbrvtNr72ta+RTqeZM2eOzax06623smTJEmprazn55JOZP38+NTU1nH766QD09fVx1VVXsXfvXmpra/nKV75ig7DPOOMMrrjiCv7H//gfTJgwgWuvvRYppbVg9DenweQ51PMr7c88/8Mf/hCA0047jUWLFrFgwQJqampYt24dkydPHnQfYsSIcexh//48zc0d7NzZSXgUZSkcDH8NYenp6eCddzbyox8tGvJ4v/rVr0gkElx55ZWfVOQYRwhCyv6OXI8NzNYBrINBeB7N2zojpdrzHZKgteP+TvGtW1Iium5O/Y1SL3yKskh5OhORr12ETKyGiXcwXZixbKB4CQmxVbYNafEiuQ1pKhpbj+8nKY6dSETKsXVB8iP5vYQOetdBzomUmkN6GDY1biIVje+6TfkJHezgRfO0gd+GxCUiomHW1XfIhy7kd/a0FK98aCw1GFMDJEpckQRY9ySUVQNf8tPT4NZHP2DauZPoNUuhu0sZYqGHTAhIe+p7ylP3fQ/KfEFGSNKewPMkKQ9Wv/IeTS++jC3aJ6WqDh4U1DXXKhbmo2xkBoVeZo8P2d/ZM9DP0yKRTPHaa68O2i5GjBjHHlasWMG9T95L39K+ouvjxo1j9OjRAIx/bDwPfv/BIyFejMOM3bv72Lx5P/v2Df5/w/GE997bQkdHH5lMGbfcsmBItZw6Ojo4+eST+eIXv8iTTz55GKQ8/nHttdeybdu2A66vWLHCJjE5RHjqBLFoaJjTeZtmVVsQiuImHHem0InbMNdt+ladjUi4AeOhDnSWKr7AdWUKTbpa4WRrErpvEQU3y0L0jPBVf34iGsM8g49Nf5tIRAquJRmO0u8nI+LiptsFJUsyFbn+WBcnsw4SCjlIZaI1lAVdd8Ox/njOGhpXKZsq15AM81GqOZtaGkJQcA/zjduZe80NCPeE5XvoWoujMyAyZdajyUPVzUggkQgVMifU1hSApNTfdTehVLEceV/FbqSkIJDgmWxlxtVLhiAyiqHk+9TLxNCY34NxnTpmKXyMGDFixDjUkBJ27Ohh8+b9dHT0Df7AcYR8Ps9bb71LRcUwMpmyj/Xsb37zG3p6evjGN77xKUkX49PEiUM0jAZqYwTMdcd9yLUMmMrWgHV9sg+51gcdUyEpdr0K89HzJpbBVst2iInUn4NCsfuVcRFy3a0MTD0H4UepXj0nq5O1tnjRu3EfK0rFm1TyBAVNfmRk5TApfE2a1kIBPO0q5DsuWlK7FLnuV24ch/2Mvl+yJ1rVNwYQI5riJ66m7rpQUeRaNSUF1UnwEgl8lPUiRJJEkEJYy4aUiqMktIUjIZQ1w/AaiSAIJYFvMlBJ8nnjnka0z8ZCZALccz2aIBqrlSmeGKq1JMdQ8Gm6TsWIESNGjCMDk0Gqubmdnp784A8cZ9ix40N2726nsnL4J3r+V7/6FbW1tfy3//bfDrFkMQ4HThyigYgUeevGo28ZZd8oz/TzLk1sgOc8K0AYpZzId18IXT8hdMiHVuAh8u03NRgEIPNAsiSA2LFKQESEhDMPazVAn6jjWDMcy4KXjJ4192Sg4ytctzC0zETkw9fP2iXR9TY8Xy2Hm3K2iFTIaO6RNh/NI0T7N3lkjKtUURC9Q1IsMTTWDkWKfCHJmGueR4DKkisQNvus2RLTjSEaQigR0qI0KFwV7QuFUD6zvrb4+EldvM8hEhJ1P9B7aMhhKNTfrsBDDJloHN4sHjFixIgR49ODySC1Zct+stnC4A8cZwgCyX/8xzukUpWUl3+yhBabNm3i1Vdf5Vvf+hbpdPoQSxjjcOAEIhpQZIkwp9MClPuSyc5kTum1i5JRkq0fjH6o1ArgZnFyU9kaH36hFXvrroXTn3Rk81CZHTQZCGVkWHH1UKndp1wrjIcT7O7Ka2RwYjckFMWdGEJknvWSjpuZdTDC1hsxNTKkJiLSmYP7iFeiPNsxzHyUiaKsaG5E62Zrk8jonvMeIshJKGiPNQ9lpQikJJCKbFgPUGFzepGTUKblzGtLhy8gKSSesbIAQWBczkx2KVMLxayjyTamCafdcxMwHqUnHAyxQSNGjBgxjn309gY0N3/6GaSOZuzb18r77++gqmrYX9XP//2//xcpZVENqBjHFo4DouHEABwM9rTf76e5F7lAFbn/UHzab4v6hVEbt30YRNYG4/pjMxNphdNovSIRZZMyLk/2GWM9MZYPIkuHezQfmlgJVKCCJBrPDb42PkNFhEl3Yip426xTScfyouUv5FQcR1EsS4CtOm4+S0OURCSriYsp2iqh5AlNc0Had/axyHphXMhMn1JZCzw1FymhNZAUpEAIiYdEIPC1ZSIphCrIpwlHSsehp0QUby70VH1PWTt8T+ILQdKTFIIgkqWQJyKiJZYmz1fZwKTUmab0byoUMJTfJ7FFI0aMGDGOZXR05Nm8+djLIHWo8c47TUDyryYZhUKBhx9+mFmzZvG5z33u0AgX47DjOCAaQ4Spf2FdnkzcBOoPkdAWBx
```

Comment:
It might be worth explaining briefly the 2 different log files. I.e that commands.log contains the commands executed and the qcodes.log contains python logging information

Reply:
Done.